test/Evolve.IntegrationTest.Cassandra/CqlStatementBuilderTest.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Evolve.Dialect.Cassandra;
+using Evolve.Migration;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Evolve.IntegrationTest.Cassandra
+{
+    public sealed class CqlStatementBuilderTest
+    {
+        [Fact]
+        public void M()
```

Comment:
M() ? What did you mean ?

Reply:
Indeed, I wrote the test thinking "I don't have a name yet I just want to try it out" then forgot to rename. Thanks spotting this.